server/controllers/apply/applicationsController.ts

```diff
@@ -34,11 +34,19 @@ export default class ApplicationsController {
     return async (req: Request, res: Response) => {
       const searchOptions = getSearchOptions<ApplicationDashboardSearchOptions>(req, ['crnOrName', 'status'])
 
-      const { pageNumber, hrefPrefix, sortBy, sortDirection } = getPaginationDetails<ApplicationSortField>(
+      const paginationDetails = getPaginationDetails<ApplicationSortField>(
         req,
         paths.applications.dashboard({}),
         searchOptions,
       )
+
+      const { pageNumber, hrefPrefix, sortBy } = paginationDetails
+      let { sortDirection } = paginationDetails
+
+      if (!sortBy || (sortBy === 'createdAt' && !sortDirection)) {
```

Comment:
Should we be setting the `sortBy` to `createdAt` here too?

Reply:
we need sortDirection  to set desc when sortBy not provided and sortBy is createdAt here

Reply:
Maybe I'm missing something here but if `sortBy` is falsy and we set `sortDirection` to `desc` what will we be sorting by?